runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldHopperPlugin.java

```diff
@@ -280,10 +286,37 @@ public void onConfigChanged(final ConfigChanged event)
 					panel.setFilterMode(config.subscriptionFilter());
 					updateList();
 					break;
+				case "customWorldCycle":
+					setCustomWorldCycle();
+					break;
 			}
 		}
 	}
 
+	private void setCustomWorldCycle()
+	{
+		String worldList = config.customWorldCycle();
+		WorldResult worldResult = worldService.getWorlds();
+		worldCycleList.clear();
+		if (worldResult != null && !worldList.isEmpty())
+		{
+			Text.fromCSV(worldList).stream()
+					.mapToInt(s ->
+					{
+						try
+						{
+							return Integer.parseInt(s);
+						}
+						catch (NumberFormatException e)
+						{
+							return 0;
+						}
+					})
+					.filter(world -> worldResult.findWorld(world) != null)
+					.forEach(world -> worldCycleList.add(worldResult.findWorld(world)));
```

Comment:
You have one too many levels of indent here.
```suggestion
				.mapToInt(s ->
				{
					try
					{
						return Integer.parseInt(s);
					}
					catch (NumberFormatException e)
					{
						return 0;
					}
				})
				.filter(world -> worldResult.findWorld(world) != null)
				.forEach(world -> worldCycleList.add(worldResult.findWorld(world)));
```